src/Windows/Avalonia.Win32/WindowImpl.AppWndProc.cs

```diff
@@ -336,6 +372,176 @@ protected virtual unsafe IntPtr AppWndProc(IntPtr hWnd, uint msg, IntPtr wParam,
                             return IntPtr.Zero;
                         }
 
+                        break;
+                    }
+                case WindowsMessage.WM_POINTERDEVICECHANGE:
+                    {
+                        //notifies about changes in the settings of a monitor that has a digitizer attached to it.
+                        //https://docs.microsoft.com/en-us/previous-versions/windows/desktop/inputmsg/wm-pointerdevicechange
+                        break;
+                    }
+                case WindowsMessage.WM_POINTERDEVICEINRANGE:
+                case WindowsMessage.WM_POINTERDEVICEOUTOFRANGE:
+                    {
+                        //notifies about proximity of pointer device to the digitizer.
```

Comment:
We should force-release explicit pointer captures when pen goes out of range

Reply:
ok